website/news.html

```diff
@@ -19,6 +19,10 @@
 <script>
 document.addEventListener('DOMContentLoaded', () => {
   const items = [
+    {
+      date: new Date('2023-06-13'),
```

Comment:
Thanks! Can you update the date for the talk at 6/9 as well? I think it should be 6/21, and re-order it?